Drivers/Src/Minions.c

```diff
@@ -22,9 +22,14 @@ void MinionsInit(void) {
 
 bool MinionsRead(Pin pin) {
     if ((kPininfoLut[pin].direction == kInput)) {
+        if (pin == kIgn1 || pin == kIgn2) {
```

Comment:
Maybe add the comment here:          // Ignition pins are negative logic, special-case them